sdk/lib/core/list.dart

```diff
@@ -674,6 +674,22 @@ abstract class List<E> implements EfficientLengthIterable<E> {
    * A range from [start] to [end] is valid if `0 <= start <= end <= len`, where
    * `len` is this list's `length`. The range starts at `start` and has length
    * `end - start`. An empty range (with `end == start`) is valid.
+   *
+   *     List<int> list1 = new List(3);
+   *     list1.fillRange(0, 2, 1);
+   *     print(list); //  [1, 1, null]
+   *
+   * Note: Usage on dynamic list throws RangeError: Invalid Value
+   *
+   *      List<int> list = new List();
+   *      list.fillRange(0, 2, 1); // throws `RangeError: Invalid Value` exception.
+   *
```

Comment:
This is an example of what not to do, which we try not to do (unless the error is particularly interesting, this one isn't).

I'd just drop this example.

Reply:
I added this keeping in mind it as an extra note or gotcha while using the fillRange method. But with your suggested comment:
` The list must be growable. `
It give user a sense that using this method might result in some error. 
I will go ahead and remove this example.